Lombiq.ContentEditors.Samples/Readme.md

```diff
@@ -0,0 +1,22 @@
+# Lombiq Content Editors for Orchard Core - Samples
+
+## About
```

Comment:
This is missing.

Reply:
Huh? What's missing exactly? It's there.

- How to create an async editor provider for a content type. Once you've enabled the module you can access the demo editor on this URL: `/Admin/ContentItemAsyncEditor/Employee`.
- How to create an async editor that can be used on the front-end instead of the Admin UI. Once you've enabled the module you can access the demo editor on this URL: `/FrontEndDemoContentItemAsyncEditor`.

Reply:
I can really only repeat what I've written initially. You're telling about the code but I'm asking about the UI. If somebody wants to first check out what kind of feature the code implements in the first place, they need some hints.